package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuxt-google-optimize",
-  "version": "0.5.4",
+  "version": "0.5.5",
```

Comment:
We shouldn't bump version for PR but release script from master will bump and update changelog